setup.py

```diff
@@ -227,21 +235,39 @@ def ctresrc(self):
 get = Downloader()
 
 _travis_build = os.environ.get("TRAVIS_BUILD")
-
+libraries = None
+cmdclass = {}
+if ctre_rpi:
```

Comment:
I feel that instead of duplicating the RoboRIO section, you should just have one section and add/remove relevant pieces depending on which platform you're on... as it seems that they're mostly the same, right?

Reply:
Mostly.  At first it was a bit too tangled and I was wary of breaking the roborio section, but I think at this point I can merge the two sections back together.

Reply:
Merged and committed.

Reply:
And fixed the trailing whitespace that was causing it to fail the black test, committed...